src/project/project.cairo

```diff
@@ -37,11 +41,54 @@ func constructor{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
     //   name(felt): Name of the collection
     //   symbol(felt): Symbol of the collection
     //   owner(felt): Owner address
+    //   proxy_admin(felt): Admin address
     // Returns:
     //   None
     ERC721.initializer(name, symbol);
     ERC721Enumerable.initializer();
     Ownable.initializer(owner);
+    CarbonableAccessControl.initializer();
```

Comment:
Do we need both Ownable AND CarbonableAccessControl?
Why not define an ADMIN role in CarbonableAccessControl? Not sure what good practices are.

Reply:
In my understanding Ownable is a simplified access control with a single role.
In case we have to manage nested roles, OZ access control lib is the solution.

More details [here](https://docs.openzeppelin.com/contracts-cairo/0.5.0/access)

Reply:
Finally owner is a specific role, for which OZ has implemented few dedicated functions (such as transfer_ownership or renounce_ownership) with according securities. Using AccessControl to define this role will just lead to reimplement all what OZ did with Ownable with no guarantee to not introduce any bugs.

Therefore, I think it is better to keep using Ownable to handle owner role.